yt/frontends/sdf/io.py

```diff
@@ -27,7 +27,7 @@ def _read_particle_coords(self, chunks, ptf):
             for obj in chunk.objs:
                 data_files.update(obj.data_files)
         assert len(data_files) == 1
-        for data_file in sorted(data_files, key=lambda x: (x.filename, x.start)):
```

Comment:
Loops in this file look broken since we sort the datafiles but don't use them at all during iteration

Reply:
Oops I didn't see this reply. Any clue on how this _should_ be fixed ?